android-whitelist.json

```
@@ -541,6 +541,11 @@
             "group": "com\\.mercadopago\\.android\\.contacts",
             "name": "contacts"
         },
+        {
+            "group": "com\\.mercadopago\\.android\\.moneytransfer",
+            "name": "contacts_moneytransfer",
+            "version": "5\\.1\\.1"
```

Porque fija y no dinamica?
```suggestion
            "version": "5\\.\\+"
```

---

Cambiado. Gracias!